utils/check_code_format.sh

```diff
@@ -24,15 +24,17 @@
 # * Add a filtering step on the files to check, with the result stored in
 #   FILES_TO_CHECK.
 
-MODIFIED_FILES=$(git diff --name-only master | grep -E ".*\.(cpp|cc|c\+\+|cxx|c|h|hpp)$")
+BASE_BRANCH=llvm_release_100
```

Comment:
Having the `BASE_BRANCH` hard-coded makes it one more thing we have to take care of when we create new branch.
I think we might use `$TRAVIS_BRANCH` (https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) :
1. pass it from `.travis.yaml`, as a command line argument to the script. OR
2. use `$TRAVIS_BRANCH` in the scrip if the env variable exists and use `master` otherwise.

Reply:
Looks good.